docs/doxygen/config.doxyfile

```diff
@@ -875,7 +879,8 @@ INPUT_ENCODING         = UTF-8
 
 FILE_PATTERNS          = *.c \
                          *.h \
-                         *.txt
+                         *.txt \
```

Comment:
Be careful leaving in the .txt format.  If we need to use text files for conventional purposes we may not want them rolled into the documentation.